keps/sig-apps/3998-job-success-completion-policy/README.md

```diff
@@ -214,6 +215,18 @@ So, the status can never switch from `SucessCriterionMet` to `Failed`.
 Additionally, once the job has `SuccessCriteriaMet=true` condition, the job definitely ends with `Complete=true` condition
 even if the lingering pods could potentially meet the failure policies.
 
+#### The scope of the SuccessCriteriaMet condition
+
+As part of this KEP we introduced the `SuccessCriteriaMet` condition scoped to
+the success policy.
+
+However, we are going to extend the scope of the condition to the scenario when
+the Job completes by reaching the `.spec.completions`, as part of fixing
+(issue #123775)[https://github.com/kubernetes/kubernetes/issues/123775].
```

Comment:
Could you update the condition transition diagram as well?

https://github.com/kubernetes/enhancements/blob/ad65a440dd579ca5cc696ebb28e03f5f93ebe9d9/keps/sig-apps/3998-job-success-completion-policy/README.md#transition-of-statusconditions

Reply:
Updated, PTAL

Reply:
Looks good to me. Thank you!